wayat/frontend/lib/features/profile/pages/edit_profile_page/edit_profile_page.dart

```diff
@@ -146,7 +147,13 @@ class _EditProfilePageState extends State<EditProfilePage> {
           } else {
             showModalBottomSheet(
                 context: context,
-                builder: (builder) => _getImageFromCameraOrGallary());
+                builder: (builder) {
+                  if (widget.platformService.isDesktop) {
```

Comment:
Maybe you can make a test to check which of these functions are called

Reply:
I tried it but it gives me an error, so I decided to delete the test.